user_docs/en/userGuide.t2t

```diff
@@ -267,6 +267,70 @@ To open NVDA's menu, press ``NVDA+n``.
 To open NVDA's general settings dialog directly, press ``NVDA+control+g``.
 Many settings screens have keystrokes to open them directly, such as ``NVDA+control+s`` for synthesizer, or ``NVDA+control+v`` for other voice options.
 
+++ Add-ons ++[Addons]
+Add-ons are programs which provide new or changed functionality for NVDA.
+Add-ons are developed by the NVDA community, or external companies and are unaffiliated with NV Access.
+
+As with any software, it is important to be confident in the developer of an add-on before using it.
+The first time the add-on store is opened, NVDA displays a warning about add-ons.
+Add-ons are not vetted by NV Access and may have unrestricted functionality and access to information.
+Press ``spacebar`` if you have read the warning and do not need to see it next time.
+Press ``enter`` to accept the warning and proceed to the Add-on Store.

```

Comment:
The OK button is not the default control, so you have to specifically tab to it to activate it:
```suggestion
Press ``tab`` to the "OK" button, then ``enter`` to accept the warning and proceed to the Add-on Store.

```

Reply:
Suggest adding the word "reach", and aren't we using markdown style now for key indications?

```suggestion
Press ``tab`` to reach the "OK" button, then ``enter`` to accept the warning and proceed to the Add-on Store.
```

Reply:
> aren't we using markdown style now for key indications?

Oh, never mind, I see this PR is against the non-existant t2t file. I guess @seanbudd or someone is going to convert it after the fact?